fluent-dom/src/overlay.js

```diff
@@ -71,8 +71,14 @@ export default function overlayElement(element, translation) {
     return;
   }
 
+  const whitelistedAttrs = element.hasAttribute('data-l10n-attrs') ?
+    element.getAttribute('data-l10n-attrs')
+      .split(',').map(attr => attr.trim()) :
+    null;
+
   for (const [name, val] of translation.attrs) {
-    if (isAttrAllowed({ name }, element)) {
+    if (isAttrAllowed({ name }, element) ||
+        (whitelistedAttrs && whitelistedAttrs.includes(name))) {
```

Comment:
You can process the list once, but move the `includes` check into `isAttrAllowed`. **If** that's what you want. It sounds like it shouldn't be possible to add those whitelisted attributes to the element's children?